src/routes/familie-overzicht/+page.svelte

```diff
@@ -0,0 +1,503 @@
+			<li>Geplande activiteiten: <strong>1</strong></li>
+		</ul>
+		<p class="card-allert-first card-allert-grey">Boeken te laat: <strong>3</strong></p>
+		<p class="card-allert-second card-allert-grey">Openstaande boeten: <strong>0</strong></p>
+	</article>
+
+	<article class="card-user">
+		<header class="card-header">
+			<h2>Benjamin</h2>
+		</header>
+		<details>
+			<summary></summary>
+			<form id="form-one">
+				<div>
+					<label for="email">Email</label>
+					<input
+						type="text"
+						id="email"
+						name="email"
+						placeholder="Email"
+						value="mark.oba@gmail.com"
+					/>
+
+					<label for="adress">Adress</label>
+					<input
+						type="text"
+						id="adress"
+						name="adress"
+						placeholder="Adress"
+						value="Librastraat 23"
+					/>
+
+					<label for="postcode">Postcode</label>
+					<input type="text" id="postcode" name="postcode" placeholder="Postcode" value="1422LP" />
+				</div>
+
+				<div class="form-content-right">
+					<label for="telefoonnummer">Telefoon nummer</label>
+					<input
+						type="text"
+						id="telefoonnummer"
+						name="telefoonnummer"
+						placeholder="Telefoon nummer"
+						value="0615609827"
+					/>
+
+					<label for="abtype">Abbonoment type</label>
+					<select id="abtype" name="abbonomenttype">
+						<option value="type1">Type 1</option>
+						<option value="type2">Type 2</option>
+						<option value="type3">Type 3</option>
+					</select>
+
+					<label for="thkleur">Thema kleur</label>
+					<select id="thkleur" name="themakleur">
+						<option value="blauw">Blauw</option>
+						<option value="rood">Rood</option>
+						<option value="groen">Groen</option>
+					</select>
+
+					<input type="submit" value="Pas toe" />
+				</div>
+			</form>
+		</details>
+		<AvatarThree />
+		<ul>
+			<li>Aantal boeken gereserveerd: <strong>7</strong></li>
+			<li>Aantal boeken geleend: <strong>0</strong></li>
+			<li>Geplande activiteiten: <strong>0</strong></li>
+		</ul>
+		<p class="card-allert-first card-allert-grey">Boeken te laat: <strong>1</strong></p>
+		<p class="card-allert-second card-allert-red">Openstaande boeten: <strong>2</strong></p>
+	</article>
+
+	<article class="card-user">
+		<header class="card-header">
+			<h2>Menno</h2>
+		</header>
+		<details>
+			<summary></summary>
+			<form id="form-one">
+				<div>
+					<label for="fname">Email</label>
+					<input
+						type="text"
+						id="email"
+						name="email"
+						placeholder="Email"
+						value="mark.oba@gmail.com"
+					/>
+
+					<label for="adress">Adress</label>
+					<input
+						type="text"
+						id="adress"
+						name="adress"
+						placeholder="Adress"
+						value="Librastraat 23"
+					/>
+
+					<label for="postcode">Postcode</label>
+					<input type="text" id="postcode" name="postcode" placeholder="Postcode" value="1422LP" />
+				</div>
+
+				<div class="form-content-right">
+					<label for="telefoonnummer">Telefoon nummer</label>
+					<input
```

Comment:
De structuur van het formulier kan netter, door bij elkaar horende velden te groeperen in een fieldset, ipv in een div.

Reply:
Ik zal hier een issue van maken!